### ISSUE TITLE
Unable to debug/run robot tests in vscode when using --argumentfile option.

### ISSUE THREAD

**Issue by newbie-ta** [robocorp/robotframework-lsp#678]
**Description**
When using an argument file, vscode tests run via the code lens option do not work.

**To Reproduce**
1. Open the repo in vscode. (Note - the workspace folder is webapi src and tests are a couple of folders down.)
2. Navigate to the .robot (present at workspace/some_folder/regression file and open it in vscode)
3. Click on "Run" or "Debug" from the CodeLens on any test in the .robot file.
4. Observe that the console gives an error about any argument that is followed by the argument file.
*Note -* The tests run when the argumentfile is not supplied but, of course, fail due to missing arguments.

**Expected behavior**
The test is run.

**Screenshots**

**Versions:**
 - OS: Windows 10
 - Robot Framework Version: 4.1.3
 - Robocorp Code Version: 0.32.0
 - Robot Framework Language Server Version: 0.46.0
 - Client Version:  VSCode 1.67.1

**Logs**
```
PS C:\git\xxx\xxx\regression>  C:; cd 'C:\git\xxx\xxx\regression'; & 'C:\Users\xxxxx\AppData\Local\Temp\rf-ls-run\run_env_00_5f7waj0d.bat' '-u' 'c:\Users\xxxxx\.vscode\extensions\robocorp.robotframework-lsp-0.47.2\src\robotframework_debug_adapter\run_robot__main__.py' '--port' '54213' '--no-debug' '--argumentfile' 'C:\git\xxxx\xxxx\regression\args-local.txt' '--pythonpath' 'c:\git\xxxx\xxxx\regression\common\libs' '--variable' 'EXECDIR:C:/git/xxx/xxx/regression' '--prerunmodifier=robotframework_debug_adapter.prerun_modifiers.FilteringTestsSuiteVisitor' 'c:\git\xxxx\xxxx\regression\api\APICache_Test.robot' 

[ ERROR ] Parsing '--pythonpath' failed: File or directory to execute does not exist.
```

**Comment by fabioz** [comment]
I believe this is fixed in the latest release `0.47.2` (in https://github.com/robocorp/robotframework-lsp/commit/120308c40e01e79d13bfc206920bd40845ef44bb).

If it still doesn't work for you after upgrading, please comment to reopen the issue.

**Comment by newbie-taa** [comment]
I have upgraded but the issue is still not fixed @fabioz . Still get a `Parsing '--pythonpath' failed: File or directory to execute does not exist.` error when trying to run the test using an argument file. 

The tests run when when removing the --argumentfile parameter but fail due to missing variables it needs from the argument file. Let me know if you need any logs.
